99_post_install.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -xe
+
+source logging.sh
+source common.sh
+
+KUBECONFIG=${KUBECONFIG:-ocp/auth/kubeconfig}
+IFCFG_INTERFACE="assets/post-install/ifcfg-interface.template"
+IFCFG_BRIDGE="assets/post-install/ifcfg-bridge.template"
+
+export bridge="${bridge:-brext}"
+
+create_bridge(){
+  echo "Deploying Bridge ${bridge}..."
+
+  FIRST_MASTER=$(oc get node -o custom-columns=IP:.status.addresses[0].address -- no-headers | head -1)
```

Comment:
```suggestion
  FIRST_MASTER=$(oc get node -o custom-columns=IP:.status.addresses[0].address --no-headers | head -1)
```
Otherwise fails for me:
```
oc get node -o custom-columns=IP:.status.addresses[0].address -- no-headers | head -1
Error from server (NotFound): nodes "no-headers" not found
```

Reply:
good catch!, fixed